src/app/pages/articles/article-detail/article.component.html

```diff
@@ -1 +1,15 @@
-<p>article works!</p>
+<div class="article" id="articleStart">
+  <h2 class="article__title">{{ article?.title }}</h2>
+  <img class="article__title-image" src="{{ article?.titleImage }}" alt="title-image">
```

Comment:
You can use this also on line 6.
```suggestion
  <img class="article__title-image" [src]="article?.titleImage" alt="title-image">
```